csharp/test/Apache.Arrow.Tests/DecimalUtilityTests.cs

```diff
@@ -121,5 +121,86 @@ public void LargeScale()
             }
 #endif
         }
+
+        public class Strings
+        {
+            [Theory]
+            [InlineData(100.12, 10, 2, "100.12")]
+            [InlineData(100.12, 8, 3, "100.120")]
+            [InlineData(100.12, 7, 4, "100.1200")]
+            [InlineData(.12, 6, 3, "0.120")]
+            [InlineData(.0012, 5, 4, "0.0012")]
+            [InlineData(-100.12, 10, 2, "-100.12")]
+            [InlineData(-100.12, 8, 3, "-100.120")]
+            [InlineData(-100.12, 7, 4, "-100.1200")]
+            [InlineData(-.12, 6, 3, "-0.120")]
+            [InlineData(-.0012, 5, 4, "-0.0012")]
+            public void FromDecimal(decimal d, int precision, int scale, string result)
```

Comment:
Can you add a test from the BigQuery scenario:

[InlineData(7.89, 76, 38,"789000000000000000000000000000000000000" )]